starcounter-include.html

```diff
@@ -114,6 +111,7 @@
          */
         StarcounterIncludePrototype.createComponent = function StarcounterIncludeCreated() {
             var starcounterInclude = this;
+            this.defaultComposition = null;
```

Comment:
Explicit composition - composition set directly from `starcounter-layout-html-editor`
Provided composition - custom composition provided in JSON by CompositionProvider
Default composition - composition found in the view HTML Import (`declarative-shadow-dom` part)
Fallback composition - a single `<slot></slot>` to display all things from light DOM in case no composition was provided

Yes, I agree:

Reply:
That's great to have solid naming for different means to provide composition, it definitely makes talking about development and testing easier. I think we should for sure have it in JSDocs, and maybe inREADME as well. But not in the way that provides a new definition of a "term" (like formerly partial etc), rather as a clear description how composition could be provided.